src/script/components/InputBar/InputBar.tsx

```diff
@@ -193,11 +194,9 @@ export const InputBar = ({
     setCurrentMentions([]);
 
     if (resetInputValue) {
-      lexicalRef.current?.update(() => {
+      editorRef.current?.update(() => {
         $getRoot().clear();
       });
-
-      setInputValue('');
```

Comment:
We never need to manually update the state anymore, it's automatically updated from lexical's update callback